src/OSPSuite.Core/Domain/Model.cs

```diff
@@ -22,6 +19,17 @@ public interface IModel : IObjectBase, IMolWeightFinder
       ///    Returns the total drug mass parameter defined in the model for the <paramref name="moleculeName"/> if available or null otherwise
       /// </summary>
       IParameter TotalDrugMassFor(string moleculeName);
+
+      /// <summary>
+      /// Returns the molecule name associated with a quantity with path <paramref name="quantityPath"/>
+      /// </summary>
+      string MoleculeNameFor(string quantityPath);
```

Comment:
I am exposing these guys because it is quite useful to retrieve the molecule name for a quantity.
Especially I am not sure how the applications parameters will be retrieve and if a moleculename will be available

That way, it is symetrical to MolWeight